utils/issue-format-bot/lib/alexa.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 William Budington
+// Copyright 2017 AJ Jordan
+//
+// This file is part of the HTTPS Everywhere issue management bot.
+//
+// The HTTPS Everywhere issue management bot is free software: you can
+// redistribute it and/or modify it under the terms of the GNU Affero
+// General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// The HTTPS Everywhere issue management bot is distributed in the
+// hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+// the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+// PURPOSE.  See the GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with the HTTPS Everywhere issue management bot.  If
+// not, see <http://www.gnu.org/licenses/>.
+
+'use strict';
+
+const request = require('request'),
+      unzip = require('unzip');
+
+// (Heavily) modified from code by @Nainish in utils/labeller. Thanks, @Hainish!
+
+// TODO make this return Promises
+// TODO test this file
+
+module.exports = function getAlexa(cb) {
+
+	const alexa = [];
+	const csvRegex = /^[0-9]+,(.+)/;
+
+	request.get('https://s3.amazonaws.com/alexa-static/top-1m.csv.zip')
```

Comment:
The one thing about this approach is that the alexa rankings are downloaded once at the beginning, and those rankings are used for the lifetime of the process.  This is not an issue in the PR labeller since it runs as a scheduled task on any new PRs, but this is meant to run continuously.

What you really want is for an issue to be labelled according to its alexa ranking *at the time of open/edit*.  @koops76 suggestion would do that for any new issue, or you could just refactor this to download the rankings every day instead.  No need to get fancy, a `setInterval` would do the trick.